source/features/add-star-repo-hotkey.js

```diff
@@ -0,0 +1,17 @@
+import select from 'select-dom';
+import {registerShortcut} from './improve-shortcut-help';
+
+export default function () {
+	registerShortcut('repos', 'g s', 'Star and unstar repository');
+	let lastKey;
+	document.addEventListener('keydown', e => {
+		if ((lastKey === 'g' && e.key === 's') || (e.key === 'g' && lastKey === 's')) {
+			const starButtonSelector = (select('div.js-social-container.starring-container').classList.contains('on')) ? 'form.starred.js-social-form > button' : 'form.unstarred.js-social-form > button';
```

Comment:
Doesn't matter for this PR, but in the future instead `classList.contains` you could use:
```js
select.exists('.js-social-container.starring-container.on')
```

Or avoid the ternary and just select both buttons with

```js
select('form.starred.js-social-form > button, form.unstarred.js-social-form > button')
```